src/app/cli/src/app.rs

```diff
@@ -156,10 +167,13 @@ pub async fn run(
         let base_catalog = base_catalog_builder.build();
 
         // Database requires extra actions:
-        let final_base_catalog = if let Some(db_config) = config.database {
+        let final_base_catalog = if let Some(db_config) = database_config {
             // Connect database and obtain a connection pool
             let catalog_with_pool = connect_database_initially(&base_catalog).await?;
 
+            // Apply migrations to the connected database
+            apply_migrations(&catalog_with_pool).await;
```

Comment:
Maybe this should be hidden in database plugin?

Reply:
Yeah, it's going to be the best place

Moved; Done